ee/frontend/mobile-replay/transformer/transformers.ts

```diff
@@ -77,8 +77,9 @@ function* ids(): Generator<number> {
     }
 }
 
-// TODO this is shared for the lifetime of the page, so a very, very long-lived session could exhaust the ids
-const idSequence = ids()
+// TODO this is shared for the lifetime of the page,
```

Comment:
Is this still true given we now periodically reset the value in https://github.com/PostHog/posthog/pull/19974/files?diff=unified&w=1#diff-358bfae230ac47334c43cd960c1e25c388c17a0fc613868c1924dfb5db85cc3fR1131-R1132

Reply:
ah, only if you squint 🙈